xbmc/Application.cpp

```diff
@@ -1054,9 +1053,12 @@ bool CApplication::Initialize()
 
   StartServices();
 
+  // GUI depends on seek handler
```

Comment:
please elaborate

Reply:
Search code for "GetSeekHandler". There's a bunch of places that call into the seek handler, including from GUI code.

Reply:
I just moved existing code. Git history might reveal more: https://github.com/garbear/xbmc/commit/8b841c3